Presentation.Web/Models/API/V2/Response/DataProcessing/DataProcessingRegistrationGeneralDataResponseDTO.cs

```diff
@@ -52,5 +52,14 @@ public class DataProcessingRegistrationGeneralDataResponseDTO
         /// UUID's of the organization entities selected as sub data processors
         /// </summary>
         public IEnumerable<ShallowOrganizationResponseDTO> SubDataProcessors { get; set; }
+
+        /// <summary>
+        /// Determines if the entity is considered valid. Validity is determined by the MainContract Validity.
+        /// </summary>
+        public bool Valid { get; set; }
+        /// <summary>
+        /// Defines the master contract for this data processor (many contracts can point to a data processor registration but only one can be the master contract)
```

Comment:
it is not a data processor.. but a data processing registration. A data processer is some entity which processes data such as STRONGMINDS or Ballerup Kommune